src/components/atoms/ActionBotton.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+import styled from "styled-components";
+
+const Button = styled.div`
```

Comment:
```suggestion
const ActionButton = styled.a`
  display: inline-block;
```

Making this an anchor tag allows the button to be exported as is without creating a separate wrapping React element.